lib/parsers/exif_parser.rb

```diff
@@ -48,22 +48,16 @@ def seek(n, seek_mode = IO::SEEK_SET)
     :left_bottom
   ]
 
-  def initialize(filetype, file_io)
-    @filetype = filetype
-    @file_io = IOExt.new(file_io)
+  def initialize(io_blob_with_exif_data)
+    @exif_io = IOExt.new(io_blob_with_exif_data)
     @exif_data = nil
     @orientation = nil
     @height = nil
     @width = nil
   end
 
-  def scan_image_exif
-    # Without the magic bytes EXIFR throws an error
-    @file_io.seek(0)
-    raw_exif_data = EXIFR::JPEG.new(@file_io) if @filetype == :jpeg
```

Comment:
if `EXIFR::JPEG` is not used anymore, than probably we can safely remove `require 'exifr/jpeg'` from the beginning of the file

Reply:
Good point. Less requires = faster loading time